hazelcast/src/main/java/com/hazelcast/internal/partition/impl/PartitionReplicaManager.java

```diff
@@ -68,6 +68,11 @@
  */
 public class PartitionReplicaManager implements PartitionReplicaVersionManager {
 
+    /**
+     * Marker that indicates a particular backup replica index requires a sync with the primary replica.
+     */
+    public static final long REQUIRES_SYNC = -1;
```

Comment:
what it does is documented but maybe we can also javaDoc why we need it with this managed context improvements.